bubble-wrap-style.yaml

```diff
@@ -2490,6 +2603,13 @@ layers:
                     outline:
                         color: [0.799,0.799,0.799]
                         width: [[14, 0], [16, 0.5px], [18, 3px]]
+            construction:
+                filter: { kind: construction }
+                draw:
+                    dashed:
```

Comment:
We *can* move the `dash` inline into the `draw` groups now, but probably outside (already large) scope of this PR. But for future, if we want to do that instead of having a separate style.

Reply:
Agree... though Tangram ES not supporting that kinda puts a damper on that / map design is broken without the dashing.

Reply:
Oh right, forgot it's not in ES yet.